dev/PushNotifications/PushNotificationReceivedEventArgs.cpp

```diff
@@ -119,5 +131,17 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
             m_handledUnpackaged = value;
         }
     }
+
+    std::string PushNotificationReceivedEventArgs::Utf16ToUtf8(_In_z_ const PCWSTR utf16)
```

Comment:
isn't PCWSTR const by default? PWSTR is non-const

Reply:
`true`